notebooks/SITCOMTN_101_FCU_investigation.ipynb

```diff
@@ -0,0 +1,1216 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        # Get TMTMount Elevation data
</pre><p>Can you add one empty line here to improve readability?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-101/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
done